### PR TITLE
Feature/910 build

### PR DESCRIPTION
**Pull request by sreejan-chowdhury**
Changes to install onlyoffice and configure it.

### CODE REVIEW

File: rpmconf/Install/Util/utilfunc.sh [hunk 1]
```diff
@@ -2488,17 +2378,41 @@ getInstallPackages() {
...
   echo ""
   echo "Installing:"
   for i in $INSTALL_PACKAGES; do
     echo "    $i"
   done
 }
 
+isOnlyofficeStandalone() {
+  onlyofficepkg=$1
+  # check if stroe is being installed too.
```
Comment (log2akshat) [inline rpmconf/Install/Util/utilfunc.sh:2397]: Typo

File: rpmconf/Install/Util/modules/install_rabbitmq.sh [hunk 2]
```diff
@@ -0,0 +1,150 @@
...
+	   echo "RabbitMQ Server installed."
+	else
+	   echo "RabbitMQ Server installation failed. ${install_op}"
+	fi
+	
+}
+
+install_on_ubuntu()
+{
+
```
Comment (dasiyogesh) [inline rpmconf/Install/Util/modules/install_rabbitmq.sh:120]: remove empty lines here and below

File: rpmconf/Install/Util/utilfunc.sh (hunk 1 above)
Comment (sreejan-chowdhury) [reply rpmconf/Install/Util/utilfunc.sh]: done

File: rpmconf/Install/Util/modules/install_rabbitmq.sh (hunk 2 above)
Comment (sreejan-chowdhury) [reply rpmconf/Install/Util/modules/install_rabbitmq.sh]: done

**Review by log2akshat — APPROVED** [review]

**Review by dasiyogesh — APPROVED** [review]

File: rpmconf/Install/zmsetup.pl [hunk 3]
```diff
@@ -7273,6 +7363,79 @@ sub applyConfig {
...
+  if (isEnabled("zimbra-onlyoffice") ) {
+    #enable preview
+    setLocalConfig ("oo_linux_install_path", "onlyoffice");
+
+    if ($configStatus{configOnlyoffice} eq "CONFIGURED") {
+      configLog("configOnlyoffice");
+      return 0;
+    }
+
+    if($configStatus{configOnlyoffice} ne "CONFIGURED" || $newinstall){
```
Comment (dasiyogesh) [inline rpmconf/Install/zmsetup.pl:7377]: add space after if and after `)`

File: rpmconf/Install/zmsetup.pl [hunk 4]
```diff
@@ -7273,6 +7363,79 @@ sub applyConfig {
...
+sub createDirForStandaloneOnlyoffice {
+
+  my (undef,undef,$uid,$gid) = getpwnam("zimbra");
+
+  my @dir_to_create = ('/opt/zimbra/index', '/opt/zimbra/store', '/opt/zimbra/data/tmp/mysql', '/opt/zimbra/mailboxd', 'opt/zimbra/common/conf');
+  foreach my $dir (@dir_to_create) {
+      eval { make_path($dir) };
+      if ($@) {
+        print "Couldn't create $dir: $@";
+      }else{
```
Comment (dasiyogesh) [inline rpmconf/Install/zmsetup.pl:7424]: add space after } and before {

File: rpmconf/Install/zmsetup.pl (hunk 3 above)
Comment (sreejan-chowdhury) [reply rpmconf/Install/zmsetup.pl]: done

File: rpmconf/Install/zmsetup.pl (hunk 4 above)
Comment (sreejan-chowdhury) [reply rpmconf/Install/zmsetup.pl]: done

**Pull request merged**